test/quantization/pt2e/test_quantize_pt2e.py

```diff
@@ -1277,6 +1281,66 @@ def validate(self, model: torch.fx.GraphModule) -> None:
             m, expected_node_list=node_list, expected_node_occurrence=node_occurrence
         )
 
+    def test_int16(self):
+        class Int16ActQuantizer(Quantizer):
+            def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
+                # using int32 to simulate int16
+                int16_qspec = QuantizationSpec(
+                    dtype=torch.int32,
```

Comment:
this says dtype=int32 not int16

Reply:
oh nevermind, I see the restriction on the values. I thought int16 is supported torch dtype but not uint16? Then why not use that dtype directly

Reply:
oh I think I can use torch.int16 here directly, not sure why I used int32 in the beginning